api-tests/plugins/i18n/content-manager/non-localized-fields.test.api.js

```diff
@@ -0,0 +1,548 @@
+'use strict';
+
+const { createStrapiInstance } = require('api-tests/strapi');
+const { createAuthRequest } = require('api-tests/request');
```

Comment:
nice job with the api tests! :) 

I saw there was another one skiped here `api-tests/plugins/i18n/content-api/content-types.test.api.js` , we can keep it or delete it but I think we could decide in this pr too!